ddtrace/internal/datadog/profiling/dd_wrapper/include/profile.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include "types.hpp"
+
+#include <atomic>
+#include <deque>
+#include <memory>
+#include <mutex>
+#include <string>
+#include <string_view>
+#include <unordered_set>
+#include <vector>
+
+extern "C"
+{
+#include "datadog/profiling.h"
+}
+
+namespace Datadog {
+
+constexpr unsigned int g_default_max_nframes = 128;
```

Comment:
Is there another python max defined (with a value of 100) ? I am not weighing in on what we should do, just trying to check what standards are defined in python.

Reply:
Correct, I did indeed have a value of 64 which would have been overridden by this default.  I've factored it slightly differently
1. Constants are now in a constants header
2. Confirmed that the default in the .py code is 64, added a note about the origin in the constants file.